net-analyzer/openvas-libraries/openvas-libraries-9.0.3.ebuild

```diff
@@ -0,0 +1,91 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_MAKEFILE_GENERATOR="emake"
+inherit cmake-utils
+MY_PN="gvm-libs"
+
+DESCRIPTION="A remote security scanner for Linux (openvas-libraries)"
+HOMEPAGE="http://www.openvas.org/"
+SRC_URI="https://github.com/greenbone/${MY_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+SLOT="0"
+LICENSE="GPL-2"
+KEYWORDS="~amd64 ~x86"
+IUSE="+extras ldap radius"
+
+DEPEND="
+	dev-libs/hiredis
+	dev-libs/libgcrypt:0=
+	dev-libs/libksba
+	dev-perl/UUID
+	net-analyzer/net-snmp
+	net-libs/gnutls:=[tools]
+	net-libs/libpcap
+	net-libs/libssh:=
+	sys-libs/zlib
+	extras? ( dev-perl/SQL-Translator )
+	ldap? ( net-nds/openldap )
+	radius? ( net-dialup/freeradius-client )"
+
+RDEPEND="
+	${DEPEND}"
+
+BDEPEND="
+	app-doc/doxygen[dot]
```

Comment:
Depend conditionally with extras? USE flag?

Reply:
You are right.Fixed.